examples/todo-jwt/src/index.ts

```diff
@@ -3,8 +3,13 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {TodoListApplication} from './application';
-import {ApplicationConfig} from '@loopback/core';
+import {ApplicationConfig, TodoListApplication} from './application';
+
+// re-exports for our benchmark, not needed for the tutorial itself
+export * from '@loopback/rest';
```

Comment:
I generated a new app and used it for comparison. A newly scaffolded app doesn't have these 4 lines, do we want to remove too?

Reply:
yeah these were there for some reason. I realize that generated apps don't look like this.  In todo example, there is a  morgan logger stuff, and an extra `tag` property in the todo model which is tested in the acceptance test.

Reply:
I was able to remove them, and adjust a few file imports elsewhere. tests runs.

Reply:
I cannot remove these benchmark exports from `todo` example, otherwise a benchmark tests dies in the CI.